app/src/main/java/com/example/parkingsystem/mvp/view/ParkingSpaceReservationView.java

```diff
@@ -78,26 +79,44 @@ public void enableButtonEnd() {
 
     @Override
     public int getParkingSpace() {
-        return Integer.parseInt(binding.inputParkingSpaceReservationPlace.getText().toString());
+        String parkingSpace = binding.inputParkingSpaceReservationPlace.getText().toString();
+        if (parkingSpace.isEmpty()) {
```

Comment:
this logic should be in the presenter, not in the view

Reply:
Done!